FlowCrypt/Controllers/Compose/Extensions/ComposeViewController+Nodes.swift

```diff
@@ -234,12 +234,23 @@ extension ComposeViewController {
             case let .imageTap(indexPath):
                 self.handleRecipientAction(with: indexPath, type: type)
             case let .select(indexPath, sender):
+                self.toggleRecipientTextField(isEnabled: false)
                 self.handleRecipientSelection(with: indexPath, type: type)
                 self.displayRecipientPopOver(with: indexPath, type: type, sender: sender)
             }
         }
     }
 
+    func toggleRecipientTextField(isEnabled: Bool) {
+        let types: [RecipientType] = [.to, .cc, .bcc]
+        for type in types {
+            self.recipientsTextField(type: type)?.isUserInteractionEnabled = isEnabled
+        }
+        if let type = selectedRecipientType, let textField = recipientsTextField(type: type) {
+            textField.becomeFirstResponder()
+        }
+    }
```

Comment:
I tested and didn't notice any change of behavior - cursor is still there and I can type new recipient.

But I've just found that if we'll hide cursor, as described in https://github.com/FlowCrypt/flowcrypt-ios/issues/2304, then users won't be able to tap `backspace` for removing selected recipient. So let's leave previous behaviour, I think `toggleRecipientTextField(isEnabled: Bool)` isn't needed

Reply:
Could you try to clean & build. It might be because of cache.
If it still happens after clean, please let me know reproduce steps.
> I tested and didn't notice any change of behavior - cursor is still there and I can type new recipient.


I think it's good idea to prevent typing when user selected recipient. User can delete recipient by clicking `Remove` button. Gmail app also prevents typing when recipient is selected.

>  So let's leave previous behaviour

Reply:
I thought maybe cursor isn't hiding only in simulator, but re-tested on device - same behavior.

To reproduce:
1) open compose screen
2) enter recipient
3) wait until public key is found (or not found)
4) click backspace
5) recipient became selected and background color has changed (which is expected), but typing cursor is still visible
6) type another recipient (first recipient still remains selected)
7) after you've entered second recipient, both recipients will try to search for public keys:
<img src="https://github.com/FlowCrypt/flowcrypt-ios/assets/1062093/399121f9-aa68-434b-870f-4b18ab83d960" width="400">

8) second recipient will finish searching, while first one will be in searching state indefinitely:
<img src="https://github.com/FlowCrypt/flowcrypt-ios/assets/1062093/9a41118c-ffad-4e75-9b0a-55cb1bc56a36" width="400">

Reply:
> I think it's good idea to prevent typing when user selected recipient. User can delete recipient by clicking Remove button. Gmail app also prevents typing when recipient is selected.

Ah, you probably mean case when there is popup with recipient actions, while I mean case when recipient is selected by tapping backspace (without popup).
Then here we should just deselect selected recipient when user starts typing, the same as Gmail app

Reply:
Aha. Got it

Reply:
I thought you were talking about state when there is a popup with recipient actions.
However fixed above issue. Please check